AUTHORS

```diff
@@ -788,3 +788,19 @@ Ritu Raj Singh <RituRajSingh878@gmail.com>
 Rajiv Ranjan Singh <42106787+iamrajiv@users.noreply.github.com>
 Vera Lozhkina <veralozhkina@gmail.com>
 adhoc-king <46354827+adhoc-king@users.noreply.github.com>
+Mikel Rouco <mikel.mrm@gmail.com>
+Oscar Gustafsson <oscar.gustafsson@gmail.com>
+damianos <damianos@semmle.com>
```

Comment:
@damianosSemmle if you want your name spelled out in the AUTHORS file let me know.

Reply:
@asmeurer Could you spell it as Damianos Christophides? Thanks!

Reply:
This was already merged. Can you send a PR fixing it. You need to edit `.mailmap` adding 

```
Damianos Christophides <damianos@semmle.com> damianos <damianos@semmle.com>
```

Then re-run `./bin/mailmap_update.py` to make sure it is correct, then `./bin/authors_update.py` to update AUTHORS, and commit both files.